### PR TITLE
[iOS] Remove artificial throttling of barcode scanning

### PR DESCRIPTION
**Pull request by manishkumar03**
# Summary
The barcode scanning using ML was being artificially throttled to scan barcodes once every 500ms. This is not only highly inefficient but not even required because the following callback is invoked at regular intervals by the system itself:

`- (void)captureOutput:(AVCaptureOutput *)captureOutput
    didOutputSampleBuffer:(CMSampleBufferRef)sampleBuffer
           fromConnection:(AVCaptureConnection *)connection`

With this change the barcode scanning using ML is now much faster and can actually make use of all the hardware power of the device.

Related: [Feature Request](https://github.com/react-native-community/react-native-camera/issues/2872)
## Test Plan
Tested by scanning a number of barcodes. The scanning was as accurate as before but much faster.

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅   |
| Android |    ❌     |

## Checklist

- [x] I have tested this on a device and a simulator
- [x] I added the documentation in `README.md`
- [x] I mentioned this change in `CHANGELOG.md`
- [x] I updated the typed files (TS and Flow)
- [x] I added a sample use of the API in the example project (`example/App.js`)

### CODE REVIEW

File: ios/RN/RNCamera.m [hunk 1]
```diff
@@ -92,7 +88,7 @@ - (id)initWithBridge:(RCTBridge *)bridge
         self.cameraId = nil;
         self.isFocusedOnPoint = NO;
         self.isExposedOnPoint = NO;
-        self.invertImageData = false;
+        self.invertImageData = true;
```
Comment (sibelius) [inline ios/RN/RNCamera.m:91]: why this change?

Comment (manishkumar03) [reply ios/RN/RNCamera.m:91]: If the scanning mode is set to `ALTERNATE`, this flag gets inverted. This change will make sure that the first time the image is scanned, this flag is set to false and thus we try to scan the image first as a normal image.
